android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/view/CompoundTextProperties.kt

```diff
@@ -57,3 +60,9 @@ enum class TextFontWeight(val fontWeight: FontWeight) {
   EXTRA_LIGHT(FontWeight.ExtraLight),
   SEMI_BOLD(FontWeight.SemiBold),
 }
+
+enum class TextCase {
+  UPPER_CASE,
+  LOWER_CASE,
+  CAMEL_CASE,
```

Comment:
**Pascal Case** is quite common as well, do we add it?

Reply:
Sure

Reply:
We can include it when we have a use case for it.